pkgs/development/python-modules/rctclient/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  #  aiohttp,
+  buildPythonPackage,
+  fetchFromGitHub,
+  #pytest-aiohttp,
+  #pytestCheckHook,
```

Comment:
No commented out code. Please get the tests to run or add a comment why they don't work.